libmultilabel/data_utils.py

```diff
@@ -135,8 +136,9 @@ def load_or_build_text_dict(
         embed_file (str): Path to a file holding pre-trained embeddings.
         embed_cache_dir (str, optional): Path to a directory for storing cached embeddings. Defaults to None.
         silent (bool, optional): Enable silent mode. Defaults to False.
-        normalize (bool, optional): Whether the word embeddings divide by `float(np.linalg.norm(vector) + 1e-6)`.
-        Defaults to False.
+        normalize_embed (bool, optional): Whether the word embeddings divide by `float(np.linalg.norm(vector) + 1e-6)`.
+            Defaults to False. We follow the normalization method here:
```

Comment:
I think this is the normalization method everyone uses? (x/|x|)  so why "follow ..." here?
What I would write is: We use the constant 1e-6 by following http://.. for an internal experiment of reproducing their results.